server/src/test/java/org/opensearch/extensions/ExtensionsOrchestratorTests.java

```diff
@@ -873,7 +983,14 @@ public void testOnIndexModule() throws Exception {
 
         transportService.start();
         transportService.acceptIncomingRequests();
-        extensionsOrchestrator.initializeServicesAndRestHandler(restController, settingsModule, transportService, clusterService, settings);
+        extensionsOrchestrator.initializeServicesAndRestHandler(
```

Comment:
Should we keep `initializeServicesAndRestHandler` in test setup to avoid duplication?

Reply:
Thats a very good point sure.